lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -285,7 +285,7 @@ defmodule LivebookWeb.SessionLive.CellComponent do
 
   defp cell_status(%{cell_view: %{validity_status: :stale}} = assigns) do
     ~H"""
-    <.status_indicator circle_class="bg-yellow-200" change_indicator={true}>
+    <.status_indicator circle_class="bg-yellow-bright-200" change_indicator={true}>
```

Comment:
We mirror the status in the bottom-right indicators and in sections panel, if you look `bg-yellow-200` you should find all places :D